libcxx/include/__expected/expected.h

```diff
@@ -1552,8 +1570,56 @@ class expected<_Tp, _Err> {
     _LIBCPP_NO_UNIQUE_ADDRESS _ErrorType __unex_;
   };
 
-  _LIBCPP_NO_UNIQUE_ADDRESS __union_t<_Err> __union_;
-  bool __has_val_;
+  struct __expected_repr {
```

Comment:
Let's just name this `__repr`. It's already inside `expected`.

Reply:
done!